nginx/templates/default.conf.jinja

```diff
@@ -30,8 +31,11 @@ server {
   root {{ webroot }};
   server_name {{ domain }} www.{{ domain }} {{ aliases|default([])|join(" ") }};
 
+  {{- before_location_config|default([])|join("\n")|indent(2) }}
```

Comment:
I'm all up for a system where i can define more location blocks. Currently, I'm jamming them into the `extra_config` pillar. Would it be possible to make this more intuitive in the declaration? or what are you expecting people to add here that can't be added in `extra_config`?

Reply:
Right now `extra_config` is the way.